README.md

```diff
@@ -1,2 +1,114 @@
-# doppler-html-editor-api
-API Rest from Doppler HTML Editor
+# Doppler HTML Editor API
+
+## Campaigns
+
+```http
+GET /accounts/{accountName}/campaigns/{campaignId}/content/design
+```
+
+Response:
+```json
+{
+	"counters": {},
+	"body": {},
+	"schemaVersion": 6
+}
+```
+
+## Thumbnails
+
+```http
+GET /accounts/{accountName}/campaigns/{campaignId}/content/thumbnail
+```
+
+Response:
+
+PNG/JPG file or redirect to image URI
+
+## Save Campaign
+
+```http
+PUT /accounts/{accountName}/campaigns/{campaignId}/content/
+```
+
+```json
+{
+	"counters": {...},
+	"body": {...},
+	"schemaVersion": 6,
+	"HTML": "",	// HTML
+	"thumbnailUrl": "" // URI Image
+}
+```
+
+---
+
+## Templates
+
+```http
+GET /accounts/{accountName}/templates/{templateId}
+```
+
+Response:
+```json
+{
+	  "counters": {},
+	  "body": {},
+	  "name": "",
+	  "schemaVersion": 6
+}
+```
+
+## Save Template
+
+```http
+PUT /accounts/{accountName}/templates/{templateId}
+```
+
+```json
+{
+	"name": "",
+	"counters": {...},
+	"body": {...},
+	"schemaVersion": 6,
+	"HTML": "",	// HTML
+	"thumbnailUrl": "" // URI Image
+}
+```
```

Comment:
In my opinion, it is a single thing:
```suggestion
\```http
PUT /accounts/{accountName}/templates/{templateId}

{
	"name": "",
	"counters": {...},
	"body": {...},
	"schemaVersion": 6,
	"HTML": "",	// HTML
	"thumbnailUrl": "" // URI Image
}
\```
```